discord/client.py

```diff
@@ -2043,17 +2043,75 @@ async def fetch_skus(self) -> list[SKU]:
         data = await self._connection.http.list_skus(self.application_id)
         return [SKU(data=s) for s in data]
 
-    async def fetch_entitlements(self) -> list[Entitlement]:
+    async def fetch_entitlements(
+        self,
+        user: Snowflake | None = None,
+        skus: list[Snowflake] | None = None,
+        before: SnowflakeTime | None = None,
+        after: SnowflakeTime | None = None,
+        limit: int | None = 100,
+        guild: Snowflake | None = None,
+        exclude_ended: bool = False,
+    ) -> EntitlementIterator:
         """|coro|
 
         Fetches the bot's entitlements.
 
         .. versionadded:: 2.5
 
+        .. versionchanged:: 2.6
+
+            Added new parameters
```

Comment:
Is this necessary?

Reply:
It can be removed, or we could update it to be more concise. 
@Dorukyum thoughts?

Reply:
`versionchanged` was never enforced and there's no breaking change here so I think we can remove this one